core/shortcut_items.ts

```diff
@@ -59,28 +57,28 @@ export function registerEscape() {
 export function registerDelete() {
   const deleteShortcut: KeyboardShortcut = {
     name: names.DELETE,
-    preconditionFn(workspace) {
-      const selected = common.getSelected();
+    preconditionFn(workspace, scope) {
```

Comment:
Even though this seems on the face of it like it should be OK, it turns out that because shortcuts can be extracted from the registry this is in fact a breaking change which has broken the keyboard experimentation plugin.

(Here and many places below in this file.)